tests/manual-test-cases/Group5-Functional-Tests/5-4-High-Availability.robot

```diff
@@ -204,17 +206,15 @@ Test
 
     # Abruptly power off the host
     Open Connection  ${curHost}  prompt=:~]
-    Login  root  e2eFunctionalTest
+    Login  root  ${NIMBUS_ESX_PASSWORD}
     ${out}=  Execute Command  poweroff -d 0 -f
     Close connection
 
     ${info}=  Run  govc vm.info \\*
     Log  ${info}
 
-    # Really not sure what better to do here?  Otherwise, vic-machine-inspect returns the old IP address... maybe some sort of power monitoring? Can I pull uptime of the system?
-    Sleep  4 minutes
-    Run VIC Machine Inspect Command
-    Wait Until Keyword Succeeds  20x  5 seconds  Run Docker Info  %{VCH-PARAMS}
+    # Wait for the VCH to come back up fully - if it's not completely reinitialized it will still report the old IP address 
+    Wait For VCH Initialization
```

Comment:
without the 4 minute sleep above... I think that this could potentially return before the system has powered off and did it's HA event right? sometimes the power off of the ESXi host takes a couple of minutes.  I think in that case, we could see the vic machine inspect and docker info return before the HA event even occurs.

Reply:
Good to know. Will add some logic to confirm that host has changed, probably by using govc events to wait for the HA event.